DOC_OWNERS.md

```diff
@@ -21,7 +21,7 @@ There are 144 owned istio.io docs.
 - [docs/examples/microservices-istio/single/index.md](https://preliminary.istio.io/latest/docs/examples/microservices-istio/single)
 - [docs/reference/glossary/index.md](https://preliminary.istio.io/latest/docs/reference/glossary)
 
-## istio/wg-environments-maintainers: 36 docs
+## istio/wg-environments-maintainers: 37 docs
```

Comment:
completely unrelated to this PR - we may want to remove this unless it has a clear purpose as it will lead to merge conflicts.